docs/merged_objects.md

```diff
@@ -224,8 +224,104 @@ Some merged objects may have some additional sample metadata columns specific to
+The following command can be used to access the PCA results:
+
+```r
+reducedDim(merged_sce, "PCA")
+```
+
+UMAP results were calculated using `scater::runUMAP()`, with the PCA results as input rather than the full gene expression matrix.
+The following command can be used to access the UMAP results:
+
+```r
+reducedDim(merged_sce, "UMAP")
+```
+
+### Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)
+
+ADT data from CITE-seq experiments, when present, is included within the merged `SingleCellExperiment` object as an "Alternative Experiment" named `"adt"`, which can be accessed with the following command:
+
+```r
+altExp(merged_sce, "adt") # adt experiment
+```
+
+Within this, the main expression matrix is again found in the `counts` assay and the normalized expression matrix is found in the `logcounts` assay.
+For each assay, each column corresponds to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponds to an antibody derived tag (ADT).
+In some cases, only some libraries in the merged object will have associated CITE-seq data.
+Libraries which do not have CITE-seq expression data are included in the assay matrices with all `NA` count values.
+Column names are again cell barcode sequences prefixed with the originating library id, e.g. `SCPCL000000-{barcode}`, and row names are the antibody targets for each ADT.
+
+Only cells which are denoted as "Keep" in the `colData(merged_sce)$adt_scpca_filter` column (as described [above](#singlecellexperiment-cell-metrics)) have normalized expression values in the `logcounts` assay, and all other cells are assigned `NA` values.
+However, as described in the {ref}`ADT data processing section <processing_information:Processed ADT data>`, normalization may fail under certain circumstances.
+If a given ADT library failed normalization, it will also have `NA` values in the merged object.
+
+The following additional per-cell data columns for the ADT data can be found in the main `colData` data frame (accessed with `colData(merged_sce)` [as above](#singlecellexperiment-cell-metrics)).
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `altexps_adt_sum`      | UMI count for CITE-seq ADTs                       |
+| `altexps_adt_detected` | Number of ADTs detected per cell (ADT count > 0 ) |
+| `altexps_adt_percent`  | Percent of `total` UMI count from ADT reads       |
+
+
+In addition, the following library-specific QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) can be found in the `colData` of the `"adt"` alternative experiment, accessed with `colData(altExp(merged_sce, "adt"))`.
```

Comment:
Should we specify that these were all calculated on each library prior to merging?